tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/transform/C2jModelToGeneratorModelTransformer.java

```diff
@@ -136,6 +172,228 @@ private static class MemberMapping {
             "Request", "SdkRequest", "CppSdkRequest"
     );
 
+    public static String capitalizeFirstLetter(String str) {
```

Comment:
when would we want to use this instead of using [convertToUpperCamel](https://github.com/aws/aws-sdk-cpp/blob/main/tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/codegeneration/cpp/CppViewHelper.java#L146-L148), infact i think you want to be using that and this is actually a bug.

Reply:
This API will work. 
In the implementation I wanted to just capitalize the first letter because I was prepending "Get" to the name passed in the json which has camel case but starts with small letter